src/components/Blockly-interface.jsx

```diff
@@ -409,15 +418,52 @@ export default function BlocklyInterface(props) {
 
   return (
     <div style={{ width: "100%", height: "100%" }}>
-      {height >= 460 && <ProgramLogos />}
+      {/* if the GPT panel is full Screen, the Logos should not be rendered */}
+      {height >= 460 && !fullScreenPanel && <ProgramLogos />}
       <div id="pageContainer" style={{ width: "100%", height: "100%" }}>
         <div
           id="blocklyDiv"
           style={{
             width: "100%",
             height: "100%",
           }}
+         
         ></div>
+        {/* if GPT panel is full screen, the Goals buttons should not be rendered */}
+        {!fullScreenPanel && <div
```

Comment:
If we aren't in full screen mode, then maybe we want to just give them the prompt box so they can edit their prompt to regenerate the goals. Essentially, instead of having a div containing "Goals", we can have the div contain the UserPromptInput component.